gprofiler/python.py

```diff
@@ -127,10 +136,19 @@ def snapshot(self) -> Mapping[int, Mapping[str, int]]:
 class PythonEbpfProfiler(PythonProfilerBase):
     PYPERF_RESOURCE = "python/pyperf/PyPerf"
     dump_signal = signal.SIGUSR2
+    dump_poll_interval = 0.1  # seconds
+    dump_poll_retries = 50  # 50 retries * dump_poll_interval = 5 seconds
```

Comment:
Instead of fixing the number of iterations fix the time limit and measure the loop, like `poll_process`.

Reply:
Doesn't really matter IMO (the timeout is only so that we have a timeout, idc if it's 5s or 10s).

But agree it looks better, changing